.github/workflows/schedule_tests.yml

```
@@ -37,6 +37,37 @@ jobs:
       - name: Run tests
         run: python tests/runtests.py -v2
 
+  pyc-only:
+    runs-on: ubuntu-latest
+    name: .pyc-only
+    continue-on-error: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+          cache: 'pip'
+      - name: Install and upgrade packaging tools
+        run: python -m pip install --upgrade pip setuptools wheel
+      - run: python -m pip install .
+      - name: Prepare site-packages
+        run: |
+          SITE_PACKAGE_ROOT=$(python -c 'import site; print(site.getsitepackages()[0])')
+          echo $SITE_PACKAGE_ROOT
+          python -m compileall -b $SITE_PACKAGE_ROOT
+          find $SITE_PACKAGE_ROOT -name '*.py' -print -exec rm {} \;
```

Untested, but could this be simplified to:
```suggestion
          find $SITE_PACKAGE_ROOT -name '*.py' -print -delete
```

---

When testing I checked both and both worked. I ended up doing it this way because it preserves POSIX compatibility. I assumed it was a GNU extension and wouldn't exist on macOS, so if we eventually introduced a macOS version of this job this line would require changes.

It turns out based on some research [1][2] that the `-delete` extension was introduced by BSD and thus exists on macOS's `find`. I confirmed this experimentally on a macOS laptop.

That's along way of saying yeah I'll make the change 👍🏻 

[1] https://unix.stackexchange.com/questions/167823/finds-exec-rm-vs-delete
[2] https://www.gnu.org/software/findutils/manual/html_mono/find.html#Using-the-_002ddelete-action